src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,76 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './Loader.module.css';
+
+// TODO: some props are shadowed from react-loader-spinner. Improve?
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
I am thinking we may want to have some EDS utility types for stuff like this. Codify all the well-known props so they don't vary unexpectedly. This one in particular might not be ideal tho, as not every component has all three sizes. Food for thought 🍔